res/css/views/rooms/_EventTile.scss

```diff
@@ -227,15 +239,6 @@ $threadInfoLineHeight: calc(2 * $font-12px); // See: _commons.scss
         background-color: $event-selected-color;
     }
 
-    &.mx_EventTile_highlight,
-    &.mx_EventTile_highlight .markdown-body {
-        color: $alert;
```

Comment:
Did we lose this completely - was that intentional?

Reply:
We don't lose it, rather it is specified globally on https://github.com/matrix-org/matrix-react-sdk/pull/8940/files#diff-bd83194d78de871bc4ad1c4e8d43102c4368481bbb90705152d3d17fea813919R28 under `.mx_EventTile` regardless of layouts.

Reply:
Ah, right - completely missed that. Maybe I shouldn't review PRs on a phone on my way to school 😅

Reply:
Have a nice day!